lib/segment/src/index/hnsw_index/tests/test_graph_connectivity.rs

```diff
@@ -96,7 +96,7 @@ fn test_graph_connectivity() {
         let links = hnsw_index
             .graph()
             .links
-            .links(point_id as PointOffsetType, 0);
+            .links_vec(point_id as PointOffsetType, 0);
```

Comment:
Are you saying that a change like this allows compile time inlining?

Reply:
For `links_vec()` — no (and probably it will make the performance worse). But it's only for tests.

For `links()` → `for_each_link()`, yes.